app/controllers/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+class UsersController < ApplicationController
+
```

Comment:
You need to add a bit more to this controller.   before_action :authorize_user and before_action :authorize_admin, and create the methods in the controller to ensure correct authorization. Refer to this link for details. https://learn.launchacademy.com/teams/boston-20/curricula/on-campus-boston-20/lesson_groups/weeks_7_&_8:_group_project/lessons/devise-authorization